packages/vxrn/src/exports/dev.ts

```diff
@@ -139,21 +137,6 @@ export const dev = async (optionsIn: VXRNOptions & { clean?: boolean }) => {
 
   app.use(router)
 
-  // TODO move these to router.get():
```

Comment:
does this still work with the other examples? Seems like we may want to keep this logic for non-expo projects

Reply:
Hmm, I'll have to try it as I haven't considered this case, thinking that Expo Go can only open expo projects anyway.

Maybe I can let the expo manifest plugin fallback to self-implemented manifest generation if it failed to load expo from user's project - haven't got it working yet, I thought I provided the necessary fields correctly but Expo Go iOS app just crash on load.